src/main/java/io/vlingo/lattice/model/object/ObjectEntity.java

```diff
@@ -34,9 +34,10 @@
  * whether formally or informally implemented.
  * @param <T> the type of persistent object
  */
-public abstract class ObjectEntity<T extends StateObject> extends GridActor<String>
+public abstract class ObjectEntity<T extends StateObject> extends StatelessGridActor
   implements PersistResultInterest, QueryResultInterest {
 
+  protected final String id;
```

Comment:
@VaughnVernon I am not sure if this can be final since there is the unidentified case in the `StateObject`. I am looking into changing this to non-final and updating it each time the persistent state is restored. WDYT?

Reply:
@VaughnVernon is this the way to go about this: https://github.com/vlingo/vlingo-lattice/pull/17/commits/2e8102f58ee44d9bd045e1764a8c3b99547d6e71